util/containerutil/frontend_test.go

```diff
@@ -242,10 +251,10 @@ func TestFrontendContainerLogs(t *testing.T) {
 			assert.NoError(t, err)
 			assert.Len(t, logs, 2)
 
-			assert.Empty(t, logs[testContainers[0]].Stdout)
+			assert.NotEmpty(t, logs[testContainers[0]].Stdout)
 			assert.NotEmpty(t, logs[testContainers[0]].Stderr)
 
-			assert.Empty(t, logs[testContainers[1]].Stdout)
+			assert.NotEmpty(t, logs[testContainers[1]].Stdout)
```

Comment:
We weren't running the container correctly for the test. Now that it actually runs... there are legit logs here.